app/controllers/sessions_controller.rb

```diff
@@ -18,7 +18,8 @@ class SessionsController < Devise::SessionsController
       timeout_callback.present?,
       timeout_callback.kind == :before,
       timeout_callback.raw_filter.is_a?(Proc),
-      timeout_callback.per_key == {:if=>[], :unless=>[]},
+      # FIXME: per_key doesn't work any more, find an alternative
```

Comment:
I thought we were still only on devise 3.1? If this isn't something we need to do any more, we should remove the `FIXME` comment.